src/encoded/schemas/donor_specific_assembly.json

```diff
@@ -0,0 +1,265 @@
+            "type": "integer",
+            "minumum": 0
+        },
+        "number_of_scaffolds": {
+            "title": "Number of Scaffolds",
+            "description": "Number of scaffolds in the assembly",
+            "type": "integer",
+            "minimum": 0
+        },
+        "percent_single_copy": {
+            "title": "Percent Single Copy",
+            "description": "Percentage of highly conserved genes with a single copy present in the assembly (BUSCO)",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 100
+        },
+        "percent_duplicated": {
+            "title": "Percent Duplicated",
+            "description": "Percentage of highly conserved genes with multiple copies present in the assembly (BUSCO)",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 100
+        },
+        "percent_fragmented": {
+            "title": "Percent Fragmented",
+            "description": "Percentage of highly conserved genes with fragmented representation in the assembly (BUSCO)",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 100
+        },
+        "percent_missing": {
+            "title": "Percent Missing",
+            "description": "Percentage of highly conserved genes that are missing from the assembly (BUSCO)",
+            "type": "integer",
+            "minimum": 0,
+            "maximum": 100
+        },
+        "ploidy": {
+            "title": "Ploidy",
+            "description": "Ploidy of assembly",
+            "type": "string",
+            "enum": [
+                "Diploid",
+                "Haploid",
+                "Aneuploid"
+            ]
+        },
+        "scaffold_l50": {
+            "title": "Scaffold L50",
+            "description": "L50 of scaffolds: the count of the smallest number of scaffolds whose total length makes up 50% of the assembly size",
+            "type": "integer",
+            "minimum": 0
+        },
+        "scaffold_n50": {
+            "title": "Scaffold N50",
+            "description": "N50 of scaffolds: the sequence length of the shortest scaffold at 50% of the total assembly length (bp)",
+            "type": "number",
+            "minimum": 0
+        },
+        "total_length": {
+            "title": "Total Length",
+            "description": "Total number of bases in the assembly (bp)",
+            "type": "number",
+            "minimum": 0
+        },
+        "total_ungapped_length": {
+            "title": "Total Ungapped Length",
+            "description": "Total ungapped length of the assembly (bp)",
+            "type": "number",
+            "minimum": 0
+        },
+        "quality_value": {
+            "title": "Quality Value",
+            "description": "Quality value of assembly",
+            "type": "number",
+            "minimum": 0,
+            "maximum": 50
+        },
+        "files": {
```

Comment:
Should this property be included on this item or would it make more sense to add dsa_assembly properties to files linking to this item?  I believe that is the pattern more consistent with other submittable items and may prove easier to manage overall though I am not up on the actual possible use cases for this item.?